### DISCUSSION TITLE
Could not find suitable device in

### DISCUSSION THREAD

**Discussion by ChronoRPG** [InoriRus/Kyty#32] [General]
Greetings 

I have a Error when launch the Worms game

Could not find suitable device in

![image](https://user-images.githubusercontent.com/106481523/170876332-ba6ae53b-ed2c-454c-a174-b597bd69b89e.png)

And this is the txt file

[_kyty.txt](https://github.com/InoriRus/Kyty/files/8793595/_kyty.txt)

**Comment by HalogenGam3r** [comment]
You need a high-end NVIDIA GPU to run it.

**Reply by Dylandrake03** [reply]
Would an RTX 2070 work or would a more power GPU be needed?

**Reply by GamingDucking** [reply]
don't think so, a GTX 1050 and later will be needed.

**Comment by InoriRus** [comment]
I'v got it. Radeon RX 580 has only 1 graphics and 2 compute queues. Emulator needs 8 compute queues.
I will redesign queue's selection mechanism to make it possible to share 1 compute queue between all compute rings.

**Reply by SoffiMeower** [reply]
> I'v got it. Radeon RX 580 has only 1 graphics and 2 compute queues. Emulator needs 8 compute queues. I will redesign queue's selection mechanism to make it possible to share 1 compute queue between all compute rings.

From what I pulled from VulkanGPUinfo, the 5700XT has 3 queue families. Let me just make a simple chart... (Queue Count, Graphics Bit, Compute Bit, Transfer bit)

1 + + +
2 - ++
2 - - +
Total: 1 graphics bit, 2 compute bits, 3 transfer bits.
So yeah, even on the latest cards, it only supports 1 graphics queue and 2 compute queues. (I checked a 6900XT, it's still 1 and 2)

**Comment by Acidcastle** [comment]
Hi any one can help me?
What is wrong?
![۲۰۲۲۱۲۰۱_۱۵۴۷۰۵](https://user-images.githubusercontent.com/118968131/205052065-90601716-4526-43fe-93e6-fdcf56a44808.jpg)
![۲۰۲۲۱۲۰۱_۱۵۴۶۳۴](https://user-images.githubusercontent.com/118968131/205052734-49feff82-75cd-4098-b879-cdf09587d517.jpg)

**Comment by Acidcastle** [comment]
fc_script.exe don't work
